pkg/storage/testhelpers.go

```diff
@@ -0,0 +1,11 @@
+package storage
+
+// DANGEROUSClearDatabaseTables clears system intakes and all cascading tables.
+// Do not use on production code, only in tests.
+func (s *Store) DANGEROUSClearDatabaseTables() error {
```

Comment:
I wonder if we should put this method in a `_test.go` file so it only gets compiled in during tests? That would ensure that it can't be used in non-test code.